typings/resin-sdk.d.ts

```diff
@@ -686,6 +671,44 @@ declare namespace ResinSdk {
 					): Promise<void>;
 					remove(nameOrId: string | number, tagKey: string): Promise<void>;
 				};
+				configVar: {
+					getAllByApplication(
+						nameOrId: string | number,
+						options?: PineOptionsFor<ApplicationConfigVariable>,
+					): Promise<ApplicationConfigVariable[]>;
+					set(
+						nameOrId: string | number,
+						key: string,
+						value: string,
+					): Promise<void>;
+					get(
+						nameOrId: string | number,
+						key: string,
+					): Promise<string | undefined>;
+					remove(
+						nameOrId: string | number,
+						key: string,
+					): Promise<string | undefined>;
+				};
+				envVar: {
+					getAllByApplication(
+						nameOrId: string | number,
+						options?: PineOptionsFor<ApplicationEnvironmentVariable>,
```

Comment:
Now that we have a generalised approach for KV pairs, should we have types for the KV model as well? We could cut down on a lot of duplication here